javaharness/java/arcs/android/AndroidArcsEnvironment.java

```diff
@@ -227,6 +227,7 @@ private void configureShell() {
                 .put("storage", "volatile://")
                 .put("manifest", dynamicManifest)
                 .put("urlMap", jsonParser.emptyObject()
+                    .put("https://$assets/", "file:////android_asset/arcs/")
```

Comment:
You can probably remove that in this patch, even if I add something similar later.

Reply:
Done